CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Bolt Next
+
+### New features
+
+* **Plugin support in `bolt.yml`** ([#1381](https://github.com/puppetlabs/bolt/pull/1381))
```

Comment:
I think the 1.38.0 changelog PR got merged before we could review it. Would you mind adding this section under it and deleting Bolt Next?

Reply:
Also, could you add an empty line after https://github.com/puppetlabs/bolt/pull/1381/commits/c257b05985abeb0492f4f69aac8ff3547e6e2066#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR27

Reply:
Updated

Reply:
Thanks!